src/sentry/static/sentry/app/views/eventsV2/querycard.tsx

```diff
@@ -40,62 +41,38 @@ class QueryCard extends React.PureComponent<Props> {
     } = this.props;
 
     return (
-      <StyledQueryCard
-        data-test-id={`card-${title}`}
-        onClick={this.handleClick}
-        to={this.props.to}
-      >
-        <QueryCardHeader>
-          <StyledTitle>
-            {title}
-            {starred && (
-              <StyledIconBookmark
-                color={theme.yellow}
-                data-test-id="is-saved-query"
-                solid
-              />
-            )}
-          </StyledTitle>
-          <StyledQueryDetail>{queryDetail}</StyledQueryDetail>
-        </QueryCardHeader>
-        <QueryCardBody>{renderGraph()}</QueryCardBody>
-        <QueryCardFooter>
-          <StyledCreator>{subtitle}</StyledCreator>
-          {renderContextMenu && renderContextMenu()}
-        </QueryCardFooter>
-      </StyledQueryCard>
+      <Link data-test-id={`card-${title}`} onClick={this.handleClick} to={this.props.to}>
+        <StyledQueryCard withHover>
+          <QueryCardHeader>
+            <StyledTitle>
+              {title}
+              {starred && (
+                <StyledIconBookmark
+                  color={theme.yellow}
+                  data-test-id="is-saved-query"
+                  solid
+                />
+              )}
+            </StyledTitle>
+            <StyledQueryDetail>{queryDetail}</StyledQueryDetail>
+          </QueryCardHeader>
+          <QueryCardBody>{renderGraph()}</QueryCardBody>
+          <QueryCardFooter>
+            <StyledCreator>{subtitle}</StyledCreator>
+            {renderContextMenu && renderContextMenu()}
+          </QueryCardFooter>
+        </StyledQueryCard>
+      </Link>
     );
   }
 }
 
-const StyledQueryCard = styled(Link)`
-  background: ${p => p.theme.white};
-  border: 1px solid ${p => p.theme.borderLight};
-  border-radius: ${p => p.theme.borderRadius};
-  display: flex;
-  align-items: stretch;
-  flex-direction: column;
+const StyledQueryCard = styled(Card)`
   justify-content: space-between;
-  transition: all 0.2s ease;
-  cursor: pointer;
-  text-align: left;
-  padding: 0;
-
+  height: 100%;
```

Comment:
Height is needed since the link is now the grid-level cell, not this, so to fill the grid we set the height to 100%.

Reply:
Assuming it's in a grid... wondering if that's too much of a constraint. Should be good for now though

Reply:
Ah yeah, good point. We can fix later if we need to